### PR TITLE
Fix center and add pointRadius, innerRef

### PR DESCRIPTION
**Pull request by msathis**
#### :boom: Breaking Changes
None.
- 

#### :rocket: Enhancements
Added pointRadius and fixed center.
-

#### :memo: Documentation

-

#### :bug: Bug Fix
Fixed center typo.
- 

#### :house: Internal

-

### CODE REVIEW

**Comment by hshoff** [comment]
LGTM thanks!

File: packages/vx-geo/src/projections/Projection.js [hunk 1]
```diff
@@ -47,14 +49,16 @@ export default function Projection({
...
-  if (center) currProjection.translate(center);
+  if (center) currProjection.center(center);
   if (rotate) currProjection.rotate(rotate);
   if (precision) currProjection.rotate(precision);
   if (fitExtent) currProjection.fitExtent(...fitExtent);
   if (fitSize) currProjection.fitSize(...fitSize);
 
   const path = geoPath().projection(currProjection);
 
+  if (pointRadius) path.pointRadius(pointRadius);
```
Comment (kachkaev) [inline packages/vx-geo/src/projections/Projection.js:60]:
```suggestion
  if (Number.isFinite(pointRadius)) path.pointRadius(pointRadius);
```

👋 folks! Do you if `pointRaidus` can be `0`? Would it be a valid input? I can change this in https://github.com/airbnb/visx/pull/1767 if that makes sense.

Comment (williaster) [reply packages/vx-geo/src/projections/Projection.js:60]: hey @kachkaev  according to the `d3-geo` docs this should be a 2-element numerical array https://d3js.org/d3-geo/projection#projection_center

that's consistent with our `@visx/geo` types / docs https://airbnb.io/visx/docs/geo#Projection_center

so I don't think `0` is valid.

Comment (kachkaev) [reply packages/vx-geo/src/projections/Projection.js:60]: I think it’s related to https://d3js.org/d3-geo/path#path_pointRadius (one number)

Comment (williaster) [reply packages/vx-geo/src/projections/Projection.js:60]: 🤦 🤦 🤦  sorry no idea how I misread that one. 100% agree with you/thanks for adding in your new PR 🙏

**Pull request merged**